docs/sql_database_diagram.rst

```diff
@@ -2,10 +2,447 @@
++----------------------------------+----------+---------------+----------------+
+| polarity                         | int      | sign          | -1/1           |
+|                                  |          | convention    |                |
+|                                  |          | for           |                |
+|                                  |          | production    |                |
+|                                  |          | and load      |                |
++----------------------------------+----------+---------------+----------------+
+| measurements_customer_api_key_id | int      | API to post   | 199            |
+|                                  |          | measurements  |                |
++----------------------------------+----------+---------------+----------------+
+
+systemsApiKeys
+------------------
+API key to retrieve systems measurements.
+
++----------------+----------------+-----------------+-------------------+
+| **Name**       | **Type**       | **Comment**     | **Example**       |
++================+================+=================+===================+
+| id             | int            | API key id      | 199               |
++----------------+----------------+-----------------+-------------------+
+| name           | chr            |                 | Measurements      |
++----------------+----------------+-----------------+-------------------+
+| apiKey         | chr            | API key value   | uuid-Measurements |
++----------------+----------------+-----------------+-------------------+
+
+todolist
+---------
+This table is empty in openstef-reference.
+
++----------------------+-------------+----------------+----------------+
+| **Name**             | **Type**    | **Comment**    | **Example**    |
++======================+=============+================+================+
+| id                   | int         | id of the job  |                |
++----------------------+-------------+----------------+----------------+
+| created              | datetime    |                |                |
++----------------------+-------------+----------------+----------------+
+| function             | chr         | functions to   |                |
+|                      |             | execute        |                |
++----------------------+-------------+----------------+----------------+
+| args                 | chr         | arguments of   |                |
+|                      |             | the functions  |                |
++----------------------+-------------+----------------+----------------+
+| inprogress           | int         |                |                |
++----------------------+-------------+----------------+----------------+
+
+Manually add a task besides those scheduled by Kubernetes. The list is
+automatically checked by Kubernetes.
+
+weatherforecastlocations
+------------------------
+Contains the locations of the weather stations. Not used in OpenSTEF.
```

Comment:
Is it actually used in Openstef?

Reply:
Yes, it is used for storing and retrieving weather data